source/system/String.ooc

```diff
@@ -114,7 +114,7 @@ String: class {
 	capitalize: func -> This {
 		match (this size) {
 			case 0 => this
-			case 1 => toUpper()
+			case 1 => this toUpper()
 			case =>
 				this[0 .. 1] toUpper() + this[1 .. -1]
```

Comment:
It looks like it takes upper case of the first character and leaves the rest as it is.